### PR TITLE
Renamed `sub-feed` to `service`

### PR DESCRIPTION
**Pull request by teizenman**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/22432
https://github.com/demisto/etc/issues/22422

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [X] 5.5.0

## Does it break backward compatibility?
   - [X] Yes
       - Further details: All are required fields, and were renamed.
   - [ ] No

### CODE REVIEW

File: Packs/FeedBambenekConsulting/Integrations/FeedBambenekConsulting/FeedBambenekConsulting.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ configuration:
   - D - Not usually reliable
   - E - Unreliable
   - F - Reliability cannot be judged
-  required: false
+  required: True
```
Comment (Itay4) [inline Packs/FeedBambenekConsulting/Integrations/FeedBambenekConsulting/FeedBambenekConsulting.yml]: ?

File: Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml [hunk 2]
```diff
@@ -37,7 +37,7 @@ configuration:
   - D - Not usually reliable
   - E - Unreliable
   - F - Reliability cannot be judged
-  required: false
+  required: True
```
Comment (Itay4) [inline Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml]: ?

File: Packs/FeedBambenekConsulting/Integrations/FeedBambenekConsulting/FeedBambenekConsulting.yml (hunk 1 above)
Comment (teizenman) [reply Packs/FeedBambenekConsulting/Integrations/FeedBambenekConsulting/FeedBambenekConsulting.yml]: This field should be `True`, it wasn't so changed it.
Also, verified it with @avidan-H

File: Packs/FeedAWS/Integrations/FeedAWS/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+- Renamed "sub-feed" to "service".
```
Comment (avidan-H) [inline Packs/FeedAWS/Integrations/FeedAWS/CHANGELOG.md]: I am pretty sure that when there is only one release note for a release section, we don't add the `-`.

Comment (avidan-H) [inline Packs/FeedAWS/Integrations/FeedAWS/CHANGELOG.md]:
```suggestion
Renamed **sub-feed** to **service**.
```

File: Packs/FeedBambenekConsulting/Integrations/FeedBambenekConsulting/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+- Renamed "sub-feed" to "service".
```
Comment (avidan-H) [inline Packs/FeedBambenekConsulting/Integrations/FeedBambenekConsulting/CHANGELOG.md]:
```suggestion
Renamed **sub-feed** to **service**.
```

Comment (avidan-H) [reply Packs/FeedBambenekConsulting/Integrations/FeedBambenekConsulting/CHANGELOG.md]: Make this same change to all the changelogs that have this line.

File: Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture.py [hunk 5]
```diff
@@ -36,7 +36,7 @@ class Client(BaseClient):
 
     def __init__(self, indicator_type: str, api_token: str, services: list, risk_rule: str = None,
                  fusion_file_path: str = None, insecure: bool = False,
-                 polling_timeout: int = 20, proxy: bool = False, threshold: int = 65, **kwargs):
+                 polling_timeout: int = 20, proxy: bool = False, threshold: int = 65):
```
Comment (avidan-H) [inline Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture.py]: Please explain your change.

File: Tests/conf.json [hunk 6]
```diff
@@ -2608,9 +2609,6 @@
             "demisto/py-ews:2.0": {
                 "memory_threshold": 150
             },
-            "demisto/python3:3.8.1.6120": {
-                "memory_threshold": 80
-            }
```
Comment (avidan-H) [inline Tests/conf.json]: Please explain this change.

**Comment by avidan-H** [comment]
I don't understand - I don't see any actual changes of any parameters or code... you just have changelogs saying you changed stuff...

**Comment by teizenman** [comment]
Branch got messed up.

new PR is: https://github.com/demisto/content/pull/6005

**Pull request closed without merging**